<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindObjectDialog</class>
 <widget class="QDialog" name="FindObjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Objects</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="FindWidget" name="find_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FindWidget</class>
   <extends>QWidget</extends>
   <header>find_widgets/find_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
